templates/components/_nav_mobile.html.twig

```diff
@@ -1,109 +1,104 @@
 <nav class="gtm__navigation__mobile l__wrapper l__hide--above-small-laptop space--20-0">
     <ul class="list__links list__links--row list__links--nav l__row--h-stretch">
         <li><a href="{{ path('homepage') }}" class="link--no-decor text--logo">En Marche !</a></li>
+        <li><a class="btn btn--pink" href="{{ path('donation_index') }}">Donner</a></li>
     </ul>
     <div class="nav-mobile">
         <input type="checkbox" id="nav-menu">
         <label for="nav-menu">Menu de navigation</label>
         <div class="nav-mobile--menu">
-            {% if not app.user %}
-                <ul>
-                    <li>
-                        <a href="{{ path('app_user_login') }}">Connexion</a>
-                    </li>
-                    <li>
-                        <a href="{{ path('app_membership_join') }}">Adhérer</a>
-                    </li>
-                </ul>
-                <hr />
+            <ul>
+                {% if not app.user %}
+                <li>
+                    <a href="{{ path('app_user_login') }}">Connexion</a>
+                </li>
+                {% endif %} {% if not has_role_adherent %}
+                <li>
+                    <a href="{{ path('app_membership_join') }}">Adhérer</a>
+                </li>
+                {% endif %}
+                <li>
+                    <a href="{{ path('donation_index') }}">Donner</a>
+                </li>
+            </ul>
+            <hr /> {% if has_role_user %}
+            <ul>
+                <li>
+                    <a href="{{ has_role_adherent ? path('app_user_profile') : path('app_user_login') }}">
+                        <div class="text--bold">{{ app.user.firstName }}</div>
+                        <div class="text--small text--gray">{{ get_user_level_label(app.user) }}</div>
+                    </a>
+                </li>
+            </ul>
+            <hr /> {% if has_role_adherent %}
+            <ul>
+                <li>
+                    <a href="{{ path('app_summary_manager_index') }}">Profil</a>
+                </li>
+                <li>
+                    <a href="{{ path('app_adherent_events') }}">Mes activités</a>
+                </li>
+            </ul>
+            {% endif %} {% if is_granted('CREATE_CITIZEN_PROJECT') %}
```

Comment:
one statement per line.

it is easier then to move "blocks", you can select the if/endif lines easily